packages/components/pages/f-contact-preferences/package.json

```diff
@@ -32,7 +32,8 @@
     "lint": "vue-cli-service lint",
     "lint:fix": "yarn lint --fix",
     "lint:style": "vue-cli-service lint:style",
-    "test": "vue-cli-service test:unit"
```

Comment:
just need to add the component script too here -  `"test-component:chrome": "cross-env-shell TEST_TYPE=component wdio ../../../../wdio-chrome.conf.js",`